lib/friendly_shipping/services/usps/rate_estimate_package_options.rb

```diff
@@ -65,6 +55,12 @@ def service_code
           service_code << 'COMMERCIAL' if commercial_pricing
           service_code.join(' ')
         end
+
+        private
+
+        def value_or_default(key, default, kwargs)
+          kwargs.key?(key) ? kwargs.delete(key) : default
+        end
```

Comment:
Ruby 3 changed the way keyword args have to be passed. This change was necessary because the default value for several of the keyword args was `true` and using this code would have broken:

`@foo = kwargs.delete(:foo) || true`

If we passed `foo: false` then `@foo` would have incorrectly been assigned `true` as its value (since `false || true` is `true`). So instead I'm checking if the key exists, and if it does, returning whatever value is there.